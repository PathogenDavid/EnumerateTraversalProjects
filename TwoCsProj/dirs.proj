<Project Sdk="Microsoft.Build.Traversal">
  <ItemGroup>
    <ProjectReference Include="Zapplication/Zapplication.csproj" />
    <ProjectReference Include="Application/Application.csproj" />
  </ItemGroup>
</Project>
