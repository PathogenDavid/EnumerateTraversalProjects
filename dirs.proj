<Project Sdk="Microsoft.Build.Traversal">
  <ItemGroup>
    <ProjectReference Include="*/dirs.proj" />
  </ItemGroup>
</Project>
