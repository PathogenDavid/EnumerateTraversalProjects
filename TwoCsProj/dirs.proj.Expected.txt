TwoCsProj/Application/Application.csproj
TwoCsProj/Zapplication/Zapplication.csproj
